bolt-ktor/src/test/kotlin/test_locally/app/actions/test.kt

```diff
@@ -27,7 +29,6 @@ fun Application.main() {
     val appConfig = AppConfig.builder()
             .slack(Slack.getInstance(slackConfig))
             .signingSecret(signingSecret)
-            .singleTeamBotToken(AuthTestMockServer.ValidToken)
```

Comment:
@WilliamBergamin This should not be changed. To me, it is a bit surprising to see the CI builds with the current change succeed but I found that Ktor v2 testing behaves a bit differently. Can you change all the tests this way?

```diff
diff --git a/bolt-ktor/src/test/kotlin/test_locally/app/commands/test.kt b/bolt-ktor/src/test/kotlin/test_locally/app/commands/test.kt
index 40f2f697..87ba6b00 100644
--- a/bolt-ktor/src/test/kotlin/test_locally/app/commands/test.kt
+++ b/bolt-ktor/src/test/kotlin/test_locally/app/commands/test.kt
@@ -20,30 +20,31 @@ import util.AuthTestMockServer
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
-const val signingSecret = "secret"
-val authTestMockServer = AuthTestMockServer()
-val slackConfig = SlackConfig()
+class KtorAppCommandTest {
+    private val signingSecret = "secret"
+    private val authTestMockServer = AuthTestMockServer()
+    private val slackConfig = SlackConfig()
 
-fun Application.main() {
-    val appConfig = AppConfig.builder()
+    // Since Ktor v2, this function override needs to be inside the test class
+    private fun Application.main() {
+        val appConfig = AppConfig.builder()
             .slack(Slack.getInstance(slackConfig))
             .signingSecret(signingSecret)
+            .singleTeamBotToken(AuthTestMockServer.ValidToken)
             .build()
-    val app = App(appConfig)
+        val app = App(appConfig)
 
-    app.command("/weather") { _, ctx ->
-        ctx.ack("This a clear sunny day!")
-    }
+        app.command("/weather") { _, ctx ->
+            ctx.ack("This a clear sunny day!")
+        }
 
-    val requestParser = SlackRequestParser(app.config())
-    routing {
-        post("/slack/events") {
-            respond(call, app.run(toBoltRequest(call, requestParser)))
+        val requestParser = SlackRequestParser(app.config())
+        routing {
+            post("/slack/events") {
+                respond(call, app.run(toBoltRequest(call, requestParser)))
+            }
         }
     }
-}
-
-class KtorAppCommandTest {
 
     @Before
     fun setUp() {
```

```kotlin
class KtorAppCommandTest {
    private val signingSecret = "secret"
    private val authTestMockServer = AuthTestMockServer()
    private val slackConfig = SlackConfig()

    // Since Ktor v2, this function override needs to be inside the test class
    private fun Application.main() {
        val appConfig = AppConfig.builder()
            .slack(Slack.getInstance(slackConfig))
            .signingSecret(signingSecret)
            .singleTeamBotToken(AuthTestMockServer.ValidToken)
            .build()
        val app = App(appConfig)

        app.command("/weather") { _, ctx ->
            ctx.ack("This a clear sunny day!")
        }

        val requestParser = SlackRequestParser(app.config())
        routing {
            post("/slack/events") {
                respond(call, app.run(toBoltRequest(call, requestParser)))
            }
        }
    }

    @Before
    fun setUp() {
        authTestMockServer.start()
        slackConfig.methodsEndpointUrlPrefix = authTestMockServer.methodsEndpointPrefix
    }

    @After
    fun tearDown() {
        authTestMockServer.stop()
    }
```